src/components/cart/components/cartItem/CartItem.js

```diff
@@ -0,0 +1,60 @@
+// Libraries
+import PropTypes from "prop-types";
+
+// lodash
+import _noop from "lodash/noop";
+
+// components
+import Icon from "../../../../commonComponents/icon";
+import PlusMinusButton from "../../../../commonComponents/plusMinusButton/PlusMinusButton";
+
+// utility
+import getFormattedAmount from "../../../../utility/getFormattedAmount";
+import getIconByType from "../../../../utility/getIconByType";
+
+// css
+import "./cartItem.css";
+
+function CartItem(props) {
+    const { item, quantity, handleIncrementClick, handleDecrementClick } = props;
+    const { id, name, type, amount, currency } = item;
+
+    return (
+        <div className="cartItem flex-space-between">
+            <div className="flex">
+                <Icon src={getIconByType(type)} alt={type} className={`h-1`} />
+                <p>{name}</p>
+            </div>
+            <div className="flex align-items-center">
+                <PlusMinusButton
+                    data-id={id}
+                    label={quantity}
+                    handleIncrement={handleIncrementClick}
+                    handleDecrement={handleDecrementClick}
+                />
+                <p>{getFormattedAmount(currency, amount * quantity)}</p>
```

Comment:
create a separate function for this amount * quantity because in future you may have more things to add like taxes